CHANGES.md

```diff
@@ -4,23 +4,21 @@
 
 ## Changes
 
-* Fix create phone session abuse key by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6134
-* Fix CLI backwards compatibility by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6125
-* Override forEachDocument() to skip the cache collection by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6144
-* Fix Not Found error when deploying function from git by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6133
-* Add required params for scheduled functions by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6148
-* Update the error message for router_domain_not_configured by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6145
-* Fix _APP_EXECUTOR_HOST for upgrades by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6141
-* Change executor hostname back to appwrite-executor by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6160
-* Fix create execution request filter from previous SDK version by @stnguyen90 in https://github.com/appwrite/appwrite/pull/6146
-* Fix: AI Assistant by @Meldiron in https://github.com/appwrite/appwrite/pull/6153
```

Comment:
I think we're missing the AI Assistant fix

Reply:
added 👍